circuit_knitting/cutting/cutqc/wire_cutting.py

```diff
@@ -970,6 +1079,12 @@ def _circuit_stripping(circuit: QuantumCircuit) -> QuantumCircuit:
     return dag_to_circuit(stripped_dag)
 
 
+@deprecate_func(
+    removal_timeline="Circuit knitting toolbox 0.8.0 release",
+    since="0.7.0",
+    package_name="circuit-knitting-toolbox",
+    additional_msg="Use the wire cutting or automated cut-finding functionality in the circuit_knitting.cutting package. ",
+)
 def _read_circuit(
```

Comment:
It's not really a problem, but I think marking all these functions that we don't even expose through our public API is unnecessary.

Reply:
Okay, I've now taken out all such warnings around private functions: [dc31577](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/527/commits/dc315775d8b77de35480026dd9fa0c62f124eeeb)